### ISSUE TITLE
Not sure what is wrong

### ISSUE THREAD

**Issue by error420** [knownunown/tpl-c900-openwrt-install#5]
$ ./extract.sh
[-] go run /home/randy/decrypt.go -in backup.bin -out backup.zz.tmp dec
package main:
decrypt.go:7:1: expected 'package', found '<'
[-] go run /home/randy/decrypt.go -in backup.zz.tmp -out backup.bin.tmp dmp
package main:
decrypt.go:7:1: expected 'package', found '<'
[-] dd if=backup.bin.tmp of=md5sum bs=1 count=16
dd: failed to open 'backup.bin.tmp': No such file or directory
[-] dd if=backup.bin.tmp of=backup.tar.tmp bs=1 skip=16
dd: failed to open 'backup.bin.tmp': No such file or directory
[-] mkdir -p data
[-] tar -xf backup.tar.tmp -C data/
tar: backup.tar.tmp: Cannot open: No such file or directory
tar: Error is not recoverable: exiting now
[-] pushd data
~/data ~
[-] go run /home/randy/decrypt.go -in ori-backup-user-config.bin -out config.zz.tmp dec
package main:
../decrypt.go:7:1: expected 'package', found '<'
[-] go run /home/randy/decrypt.go -in config.zz.tmp -out config.xml dmp
package main:
../decrypt.go:7:1: expected 'package', found '<'
[-] popd
~
[-] rm ./**.tmp
rm: cannot remove './**.tmp': No such file or directory

**Comment by tnytown** [comment]
Download the entire repository from github (as a zip) and extract it to its own folder. Do not modify any of the shell scripts or `decrypt.go`.

**Comment by error420** [comment]
Thank you so much! Everything worked after I worked directly out of the original directory like you recommended. I was able to do this on USA version!

BusyBox v1.19.4 (2016-11-29 08:58:35 CST) built-in shell (ash)
Enter 'help' for a list of built-in commands.

     MM           NM                    MMMMMMM          M       M
   $MMMMM        MMMMM                MMMMMMMMMMM      MMM     MMM
  MMMMMMMM     MM MMMMM.              MMMMM:MMMMMM:   MMMM   MMMMM
MMMM= MMMMMM  MMM   MMMM       MMMMM   MMMM  MMMMMM   MMMM  MMMMM'
MMMM=  MMMMM MMMM    MM       MMMMM    MMMM    MMMM   MMMMNMMMMM
MMMM=   MMMM  MMMMM          MMMMM     MMMM    MMMM   MMMMMMMM
MMMM=   MMMM   MMMMMM       MMMMM      MMMM    MMMM   MMMMMMMMM
MMMM=   MMMM     MMMMM,    NMMMMMMMM   MMMM    MMMM   MMMMMMMMMMM
MMMM=   MMMM      MMMMMM   MMMMMMMM    MMMM    MMMM   MMMM  MMMMMM
MMMM=   MMMM   MM    MMMM    MMMM      MMMM    MMMM   MMMM    MMMM
MMMM$ ,MMMMM  MMMMM  MMMM    MMM       MMMM   MMMMM   MMMM    MMMM
  MMMMMMM:      MMMMMMM     M         MMMMMMMMMMMM  MMMMMMM MMMMMMM
    MMMMMM       MMMMN     M           MMMMMMMMM      MMMM    MMMM
     MMMM          M                    MMMMMMM        M       M
       M
 ---------------------------------------------------------------
   For those about to rock... (Attitude Adjustment, unknown)
 ---------------------------------------------------------------
root@Archer_C900:~#
root@Archer_C900:~#
root@Archer_C900:~#
root@Archer_C900:~# scp
-ash: scp: not found
root@Archer_C900:~#                                                                                                                                                                                                                



Thank you so much! I only had one major problem, no SCP on this version. I was able to do the following to get the firmware over:

PC:~$ cat openwrt-ath79-generic-tplink_archer-c2-v3-squashfs-sysupgrade.bin | ssh tplinkcloudusername@192.168.1.1 "cat > remote"

Then I did vim /etc/config/network to add gateway and dns.
Then /etc/init.d/network restart
Then root@OpenWrt:~# opkg update
Finally opkg install luci

Thanks again!
